descriptions/Engine.AppGameKit.md

```diff
@@ -0,0 +1 @@
+[**AppGameKit**](https://www.appgamekit.com/) is a game engine that offers a high level coding and programming tool which aims to be easy for beginners to learn.
```

Comment:
this will be even better
```suggestion
[**AppGameKit**](https://www.appgamekit.com/) is a game engine developed by "The Game Creators" which aims to be easy for beginners to learn.
```